presto-tests/src/main/java/com/facebook/presto/tests/datatype/CreateAndInsertDataSetup.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
I had that thought too, but weren't sure what else I could put there. Any suggestions?